charts/aws-vpc-cni/values.yaml

```diff
@@ -22,8 +22,10 @@ init:
     ENABLE_IPv6: "false"
   securityContext:
     privileged: true
+  resources:
```

Comment:
I'm not sure that specifying specific init container specific resources is worth it, or a good idea, as the idea is that the node needs to reserve enough resources for the actual containers. I am not sure how the node handles when the init container requires more or less resources than the actual containers that will run, so let's leave this as is.

Reply:
Kubernetes scheduling deals with this in a [well defined way](https://kubernetes.io/docs/concepts/workloads/pods/init-containers/#resource-sharing-within-containers). The current implementation is misleading to the user and although I don't think it actually matters for most people I think it should be customisable as I've configured here (if I wasn't aiming for a no-op I'd have made it fully explicit instead of using `default`).

Reply:
Got it. Having the extra config value is fine, let's just use `resources: {}` as below